addons/survey/models/survey_question.py

```diff
@@ -170,15 +170,17 @@ def default_get(self, fields):
             'All "Is a scored question = True" and "Question Type: Date" questions need an answer')
     ]
 
-    @api.onchange('validation_email')
-    def _onchange_validation_email(self):
-        if self.validation_email:
-            self.validation_required = False
+    @api.depends('validation_email')
+    def _compute_validation_required(self):
+        for question in self:
+            if question.validation_required is None or question.validation_email:
```

Comment:
Try to remove is None + check why required to False if we use an email. A void email is invalid, so why resetting it ?